utils.groovy

```diff
@@ -217,6 +217,25 @@ def bump_builds_json(stream, buildid, arch, s3_stream_dir, acl) {
     }
 }
 
+// Create a COSA remote session, which is usually used to
+// build on a different architecture.
+//
+// Available parameters:
+//    expiration: string that represents a golang duration for how
+//                long the container should last (i.e. 4h, 30m)
+//    image: string that represents the container image to pull
+//    workdir: string that represents the in container working directory
+def createCosaRemoteSession(params = [:]) {
```

Comment:
Aside: this (and a bunch of other functions in this file) would be a good candidate to put in coreos-ci-lib.

Reply:
yep - for another day